lib/datalog/roletester/build.rs

```diff
@@ -0,0 +1,5 @@
+use std::io::Result;
+fn main() -> Result<()> {
```

Comment:
```suggestion
use std::io::Result;

fn main() -> Result<()> {
```

Reply:
I generally advise running `cargo fmt` and `cargo clippy` on all your code which should catch this if I am not mistaken.